### ISSUE TITLE
arm v7 b.w instructions now decoding as vst4.32 (with ghidra 9.2)

### ISSUE THREAD

**Issue by marcushall42** [NationalSecurityAgency/ghidra#2722]
**Describe the bug**
byte sequence c2 f4 b4 bf used to decode (with ghidra 9.1.1) as:
    b.w [address]
now decodes (with ghidra 9.2) as:
    vst4.32 {d27[],d29[],d31[],d1[]},[r2@64],r4

**To Reproduce**
Upgrade to ghidra 9.2 and re-disassemble.

**Environment (please complete the following information):**
 - OS: linux
 - Java Version: ? Don't recall..
 - Ghidra Version: 9.2

**Additional context**
This file is language ARM:LE:32:v7 (1.103) (after running ghidra 9.2, was 1.102 under ghidra 9.1.1)

Other b.w instructions get disassembled to vst4.8, and potentially other wrong instructions.  These instructions get tagged with "Warning [Unimplemented Pcode]: Instruction pcode is unimplemented: vst4.32"

**Comment by mumbel** [comment]
I think this is a known issue, have you tried 9.2.2?

**Comment by marcushall42** [comment]
Unfortunately the binaries are controlled by our customer.  I did search the issues, but certainly may have missed it.  I will ask if they can further update to 9.2.2, which should be much easier if it doesn't require database format updates.

**Comment by marcushall42** [comment]
Okay, I got an install of 9.2.2, and all of the instances that I started seeing with 9.2 were fixed.
But, there was one instance in our binary where the bytes
    84 31 4f f4
appear to be invoked as an ARM instruction.  This was disassembled into:
    vst4.32 {d25,d25,d25,d25},[pc],r4
along with a bookmark warning of Unimplemented Pcode.

I was able to go back to ghidra 7.4.2 and saw the same disassembly, so this is certainly not anything new.
